src/helpers/validateSignUp.js

```diff
@@ -0,0 +1,17 @@
+const validatePasswordsEquality = (password, repeatedPassword) =>
+  password === repeatedPassword;
+
+const checkPassword = (password) => {
+  // at least one number, one lowercase and one uppercase letter
+  // at least six characters
+  const re = /(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{6,}/;
```

Comment:
i love regex :)

Reply:
I hate ti :)

Reply:
Obviously I copied that from some where.